enonic/build.gradle

```diff
@@ -0,0 +1,106 @@
+plugins {
+    id 'java'
+    id 'com.moowork.node' version '1.3.1'
+    id 'com.enonic.xp.app' version '2.0.0'
+}
+
+app {
+    name = "${appName}"
+    displayName = "${appDisplayName}"
+    vendorName = "${vendorName}"
+    vendorUrl = "${vendorUrl}"
+    systemVersion = "${xpVersion}"
+}
+
+dependencies {
+    //compile "com.enonic.xp:script-api:${xpVersion}"
+    include "com.enonic.xp:lib-event:${xpVersion}"
+    include "com.enonic.xp:lib-content:${xpVersion}"
+    include "com.enonic.xp:lib-portal:${xpVersion}"
+    include "com.enonic.lib:lib-thymeleaf:2.0.0"
+    //include "com.enonic.xp:lib-auth:${xpVersion}"
+    //include "com.enonic.xp:lib-context:${xpVersion}"
+    //include "com.enonic.xp:lib-i18n:${xpVersion}"
+    //include "com.enonic.xp:lib-io:${xpVersion}"
+    //include "com.enonic.xp:lib-mail:${xpVersion}"
+    //include "com.enonic.xp:lib-repo:${xpVersion}"
+    //include "com.enonic.xp:lib-websocket:${xpVersion}"
+    //include "com.enonic.lib:lib-mustache:2.0.0"
+    //include "com.enonic.lib:lib-util:2.0.0"
+}
```

Comment:
In this case we can keep these commented lines. This file list all libraries Enonic ships with and according to how project develop we might need or not these packages.